app/menu/[slug]/page.tsx

```
@@ -0,0 +1,85 @@
+    url: `${SITE_URL}/menu/${params.slug}`,
+    offers: {
+      "@type": "Offer",
+      priceCurrency: "NPR",
+      price: item.price || "0",
+      availability: "https://schema.org/InStock",
+    },
+  };
+
+  return (
+    <main style={{ padding: 20 }}>
+      <script type="application/ld+json" dangerouslySetInnerHTML={{ __html: JSON.stringify(menuItemSchema) }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)